src/System.Text.JsonLab.Dynamic/System/Text/Json/Deserializer.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Buffers.Text;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Reflection;
+using System.Reflection.Emit;
+
+namespace System.Text.JsonLab
+{
+    public partial class JsonDynamicObject : DynamicObject, IBufferFormattable
```

Comment:
Why is the deserializer a DynamicObject? I would think that it's going to deserialize to a strongly typed object, i.e. no need for dynamic.

Reply:
It was already a DynamicObject. I just left it as is for now.